tsconfig.module.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "./tsconfig",
+    "compilerOptions": {
+      "target": "esnext",
+      "outDir": "build/module",
+      "module": "esnext"
+    },
+    "exclude": [
+      "node_modules/**"
+    ]
```

Comment:
so it means for this build you want to re-add the `tests/*` ? Why ?
Is it necessary to add this exclude in this tsconfig file?

Reply:
Good find, this was not intentional.